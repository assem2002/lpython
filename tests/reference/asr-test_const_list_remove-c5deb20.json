{
    "basename": "asr-test_const_list_remove-c5deb20",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_const_list_remove.py",
    "infile_hash": "e2bfdbf86ced2fae5e85037b6ea8479c1b73ab9b11b21c259739f8c1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_const_list_remove-c5deb20.stderr",
    "stderr_hash": "fdc0593ffd5807da577db67b457e53749498f6d8294ba1d0dc6aac49",
    "returncode": 2
}