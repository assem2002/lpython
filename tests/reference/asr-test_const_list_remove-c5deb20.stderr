semantic error: cannot remove element from a const list
 --> tests/errors/test_const_list_remove.py:7:5
  |
7 |     CONST_INTEGER_LIST.remove(1)
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
